test/deploy/kubernetes/manual/destroy.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+SCRIPT_DIR=$(dirname $0)
+MAXNODES="3"
+NAMESPACE="cortx"
+
+function print_header {
+    echo -e "---------------------------------------------------"
+    echo -e "$1"
+    echo -e "---------------------------------------------------"
+}
+
+# Delete Storage PODs
+for NODE_INDEX in $(seq 1 $MAXNODES); do
+    NODE_NAME="node"$NODE_INDEX;
+    print_header "Deleting POD for - $NODE_NAME"
+    NODE_POD=$SCRIPT_DIR/provisioner-pods/deployment_storage_$NODE_NAME.yaml;
+    kubectl delete -f $NODE_POD --namespace $NAMESPACE;
+    sleep 5;
```

Comment:
[SUGGESTION] instead of blind sleep, keep poling, this will avoid unnecessary sleep time and avoid further issues if delete takes more than 5 seconds of time.

        echo "Deleting the namespace: $NAMESPACE"
        try=1; max_tries=10
        until kubectl get namespace | grep $NAMESPACE >/dev/null 2>&1
        do
            if [[ "$try" -gt "$max_tries" ]]; then
                echo "
                      ERROR: Could not delete namespace after $max_tries attempts.\
                           Delete is manually and retry, exiting."
                exit 1
            fi
            try=$(( $try + 1 ))
        done